internal/pkg/service/stream/storage/model/file.go

```diff
@@ -75,16 +74,14 @@ func (f File) LastStateChange() utctime.UTCTime {
 	}
 }
 
-func (f File) Telemetry(clk clock.Clock) []attribute.KeyValue {
+func (f File) Telemetry() []attribute.KeyValue {
 	lastStateChange := f.LastStateChange().Time()
 	return []attribute.KeyValue{
 		attribute.String("project.id", f.ProjectID.String()),
 		attribute.String("branch.id", f.BranchID.String()),
 		attribute.String("source.id", f.SourceID.String()),
 		attribute.String("sink.id", f.SinkID.String()),
 		attribute.String("file.id", f.FileID.String()),
-		attribute.String("file.age", clk.Since(lastStateChange).String()),
-		attribute.String("file.state", f.State.String()),
```

Comment:
Because ctx attrs are set before operation and not updated, ... so old state was confusing in the output logs.

The `state` is useful in the `cleanup` package, because it determines the file expiration,
but in operators, we don't need it (or we have to manually update ctxattrs after a successful operation).

Reply:
We had discussion here:
https://keboolaglobal.slack.com/archives/C055WKPHYSD/p1724743220728909?thread_ts=1724739535.521059&cid=C055WKPHYSD
TL;DR
the state is changed in all `operators` on `coordinator` or `readernode` level, but is read from mirror, which is older than current state of slice/file, therefore we do not want to propagate the `state`.

We had also discussion on `age` and the reason behind it is, that the `age` is important only during cleanup of  `file` but not important when changing file states and makes noise in logging. Therefore the `cleanup/node` adds this attributes and simplyfies dependendencies of `keboola.bridge` module.